Framework/Algorithms/inc/MantidAlgorithms/WorkspaceJoiners.h

```diff
@@ -56,6 +56,8 @@ class DLLExport WorkspaceJoiners : public API::Algorithm {
 
   using Mantid::API::Algorithm::validateInputs;
   void validateInputs(const API::MatrixWorkspace &ws1,
```

Comment:
We can collapse these into a single method with a default parameter:
```
void validateInputs(const API::MatrixWorkspace &ws1,  const API::MatrixWorkspace &ws2, const bool checkBinning=true);
```

rather than needing two methods.